### ISSUE TITLE
the results on pascal voc look strange

### ISSUE THREAD

**Issue by CuberrChen** [hzhupku/SemiSeg-AEL#12]
according to table 6，supervised baseline with different label ratio has achieved good results. and none of the other methods seemed to work as their paper presented.
it is strange that the results are very different between with others paper.
for example, in PixelSSL, there are significant decrease of miou with diff label ratio.
[PixelSSL](https://github.com/ZHKKKe/PixelSSL/tree/master/task/sseg)

**Comment by YanFangCS** [comment]
I also found something strange in low data ratio split  scenarios with pascal voc augmentation dataset, the results of my reproduce experiments can't be as  well as reported in the paper.  Hope author release detailed parameter settings used in his experiment.

**Comment by qdd1234** [comment]
Hi, Do you find the code in the following picture? I only find the code to print this information, But I can't find a real use for it
![image](https://user-images.githubusercontent.com/68497831/145541670-00ea71f4-d9e6-4b00-a24e-ce9ca57516b6.png)
![image](https://user-images.githubusercontent.com/68497831/145542026-f08cf393-347a-497c-a9b6-9cfac33e92c1.png)

**Comment by YanFangCS** [comment]
@qdd1234  The author uses it in the following lines. You can find it in the file train.py from line 228 to line 231.
```
    for step in range(len(data_loader_unsup)):
        i_iter = epoch * len(data_loader_unsup) + step
        lr = lr_scheduler.get_lr()
        lr_scheduler.step()
        if acp or acm:
            conf = 1 - class_criterion[0]
            conf = conf[target_cat]
            conf = (conf**0.5).numpy()
            conf = np.exp(conf)/np.sum(np.exp(conf))
            query_cat = []
            for rc_idx in range(num_cat):
                query_cat.append(np.random.choice(target_cat, p=conf))
            query_cat = list(set(query_cat))
```

**Comment by CuberrChen** [comment]
I found that the reason is that the number of times the different implementation traverses the labeled data is different. In PixelSSL, they traverses the labeled data 40 epoch(times) in all partition. but in this work, it traverses the labeled data according to total iters. Therefore, the number of times label data is seen under different partitions is different.

I don't know which is more fair / appropriate...
